stories/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,385 @@
+      </AccordionBody>
+
+      <AccordionHeader
+        active={collapseOpen3}
+        onToggle={() => setCollapseOpen3(!collapseOpen3)}
+      >
+        Collapse header #3
+      </AccordionHeader>
+      <AccordionBody active={collapseOpen3}>
+        Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf moon tempor,
+        sunt aliqua put a bird on it squid single-origin coffee nulla assumenda
+        shoreditch et. Nihil anim keffiyeh helvetica, craft beer labore wes
+        anderson cred nesciunt sapiente ea proident.
+      </AccordionBody>
+    </Accordion>
+  );
+};
+
+AccordionAnnidati.storyName = 'Accordion annidati';
+
+export const HeaderAttivi = () => {
+  const [collapseOpen1, setCollapseOpen1] = useState(true);
+  const [collapseOpen2, setCollapseOpen2] = useState(false);
+  const [collapseOpen3, setCollapseOpen3] = useState(false);
+  return (
+    <Accordion background='active'>
+      <AccordionItem>
+        <AccordionHeader
+          active={collapseOpen1}
+          onToggle={() => setCollapseOpen1(!collapseOpen1)}
+        >
+          Elemento Accordion #1
+        </AccordionHeader>
+        <AccordionBody active={collapseOpen1}>
+          Vestibulum hendrerit ultrices nibh, sed pharetra lacus ultrices eget.
+          Morbi et ipsum et sapien dapibus facilisis. Integer eget semper nibh.
+          Proin enim nulla, egestas ac rutrum eget, ullamcorper nec turpis.
+        </AccordionBody>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader
+          active={collapseOpen2}
+          onToggle={() => setCollapseOpen2(!collapseOpen2)}
+        >
+          Elemento Accordion #2
+        </AccordionHeader>
+        <AccordionBody active={collapseOpen2}>
+          Ad vegan excepteur butcher vice lomo. Leggings occaecat craft beer
+          farm-to-table, raw denim aesthetic synth nesciunt you probably
+          haven&apos;t heard of them accusamus labore sustainable VHS.
+        </AccordionBody>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader
+          active={collapseOpen3}
+          onToggle={() => setCollapseOpen3(!collapseOpen3)}
+        >
+          Elemento Accordion #3
+        </AccordionHeader>
+        <AccordionBody active={collapseOpen3}>
+          Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf moon tempor,
+          sunt aliqua put a bird on it squid single-origin coffee nulla
+          assumenda shoreditch et. Nihil anim keffiyeh helvetica, craft beer
+          labore wes anderson cred nesciunt sapiente ea proident.
+        </AccordionBody>
+      </AccordionItem>
+    </Accordion>
+  );
+};
+
+HeaderAttivi.storyName = 'Header Attivi - sfondo primario';
+
+export const HeaderHover = () => {
+  const [collapseOpen1, setCollapseOpen1] = useState(true);
+  const [collapseOpen2, setCollapseOpen2] = useState(false);
+  const [collapseOpen3, setCollapseOpen3] = useState(false);
```

Comment:
Should be mutually managed, see https://italia.github.io/bootstrap-italia/docs/componenti/accordion/#sfondo-primario